package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "scripts": {
+      "build": "webpack",
+      "start": "webpack-dev-server -d --hot --open"
```

Comment:
you can use webpack dev server configuration directly in weback.config.js file. Let's discuss

Reply:
Solved Sr.